### DISCUSSION TITLE
Flame shape

### DISCUSSION THREAD

**Discussion by ashutoshtdceiitm** [firemodels/fds#13260] [Q&A]
Hi,
I want to analyze instantaneous flame shapes. 
1. Is there a way to extract HRRPUV in .csv for all the locations (not just a point of an array) in the domain?
2. Is the cutoff value of HHRPUV (200 kW/m3) used to estimate flame shape during smokeview rendering applicable universally?
Regards
Ashutosh

**Comment by TristanHehnen** [comment]
You could use 3D slices, together with the fdsreader Python module. You'll get the data as mulit-dimensional NumPy array. Then, you can process it however you like.

https://firedynamics.github.io/LectureFireSimulation/content/tools/03_analysis/02_fdsreader.html

**Reply by ashutoshtdceiitm** [reply]
Hi,
Do I need to write a line beforehand, or do these data accumulate on their own?
Thanks

**Reply by mcgratta** [reply]
You need to specify this kind of output.

**Reply by drjfloyd** [reply]
22.4 Animated Planar Slices: The SLCF Namelist Group

**Reply by ashutoshtdceiitm** [reply]
SLCF can fetch the data on planes but I need to get HRRPUV throughout the domain. What exactly should I specify in the input file

**Reply by TristanHehnen** [reply]
Please open the [FDS User's Guide](https://github.com/firemodels/fds/releases/download/FDS-6.9.1/FDS_User_Guide.pdf) and search and read the section that Jason pointed you to in his message right above: "22.4 Animated Planar Slices: The SLCF Namelist Group".

**Reply by drjfloyd** [reply]
![image](https://github.com/user-attachments/assets/21ea5ffe-8293-4d8e-9597-fade5e4d3f2d)

**Reply by ashutoshtdceiitm** [reply]
Okay got it. Thanks a lot.
